doc/operation-and-maintenance/Mongoose-metrics.md

```diff
@@ -1,149 +1,217 @@
+| `[Host, unset_presence_hook]` | spiral | A user disconnects or sends an `unavailable` presence. |
 
-## Backend metrics description
+### Presences & rosters
 
-The existence of this kind of metrics is driven by the status of the corresponding module.
-For example if mod_roster is enabled, the relevant `backend` metrics for this module are created and updated.
-This kind of metrics are histograms and contain information about the execution time of certain functions from the backend module.
+| Name | Type | Description (when it gets incremented) |
+| ---- | ---- | -------------------------------------- |
+| `[Host, modPresenceSubscriptions]` | spiral | Presence subscription is processed. |
+| `[Host, modPresenceUnsubscriptions]` | spiral | Presence unsubscription is processed. |
+| `[Host, modRosterGets]` | spiral | User's roster is fetched. |
+| `[Host, modRosterPush]` | spiral | A roster update is pushed to a single session. |
+| `[Host, modRosterSets]` | spiral | User's roster is updated. |
 
-The only `backend` metrics which are always present are related to the authentication module as this one is always enabled.
+### Privacy lists
 
-## Data metrics description
+| Name | Type | Description (when it gets incremented) |
+| ---- | ---- | -------------------------------------- |
+| `[Host, modPrivacyGets]` | spiral | IQ privacy `get` is processed. |
+| `[Host, modPrivacyPush]` | spiral | Privacy list update is sent to a single session. |
+| `[Host, modPrivacySets]` | spiral | IQ privacy `set` is processed. |
+| `[Host, modPrivacySetsActive]` | spiral | Active privacy list is changed. |
+| `[Host, modPrivacySetsDefault]` | spiral | Default privacy list is changed. |
+| `[Host, modPrivacyStanzaAll]` | spiral | A packet is checked against the privacy list. |
+| `[Host, modPrivacyStanzaDenied]` | spiral | Privacy list check resulted in `deny`. |
+| `[Host, modPrivacyStanzaBlocked]` | spiral | Privacy list check resulted in `block`. |
+
+### Other
+
+| Name | Type | Description (when it gets incremented) |
+| ---- | ---- | -------------------------------------- |
+| `[Host, sessionAuthAnonymous]` | spiral | A client authenticates anonymously. |
+| `[Host, sessionAuthFails]` | spiral | A client failed to authenticate. |
+| `[Host, sessionCount]` | counter | Number of active sessions. |
+| `[Host, sessionLogouts]` | spiral | A client session is closed. |
+| `[Host, sessionSuccessfulLogins]` | spiral | A client session is opened. |
+| `[Host, xmppErrorIq]` | spiral | An `error` IQ is sent to a client. |
+| `[Host, xmppErrorMessage]` | spiral | An `error` message is sent to a client. |
+| `[Host, xmppErrorPresence]` | spiral | An `error` presence is sent to a client. |
+| `[Host, xmppErrorTotal]` | spiral | A stanza with `error` type is routed. |
+| `[Host, xmppMessageBounced]` | spiral | A `service-unavailable` error is sent, because message recipient if offline. |
```

Comment:
A `service-unavailable` error is sent, because the message recipient if offline.